.ci/pytorch/common_utils.sh

```diff
@@ -82,13 +82,22 @@ function pip_install_whl() {
 function pip_install() {
   # retry 3 times
   # old versions of pip don't have the "--progress-bar" flag
-  pip install --progress-bar off "$@" || pip install --progress-bar off "$@" || pip install --progress-bar off "$@" ||\
-  pip install "$@" || pip install "$@" || pip install "$@"
+  if [[ "$BUILD_ENVIRONMENT" != *s390x* ]] ; then
+    pip install --progress-bar off "$@" || pip install --progress-bar off "$@" || pip install --progress-bar off "$@" ||\
+    pip install "$@" || pip install "$@" || pip install "$@"
+  else
+    pip3 install --progress-bar off "$@" || pip3 install --progress-bar off "$@" || pip3 install --progress-bar off "$@" ||\
+    pip3 install "$@" || pip3 install "$@" || pip3 install "$@"
```

Comment:
Please submit separate PR (or dev infra can take care of it), as change is not specific to S390, we can use pip3 everywhere.

Reply:
https://github.com/pytorch/pytorch/pull/140185